apps/web/src/views/AddLiquidityV3/formViews/V3FormView/components/RangeSelector.tsx

```diff
@@ -47,37 +47,44 @@ export default function RangeSelector({
   const leftValue = useMemo(() => {
     if (ticksAtLimit[isSorted ? Bound.LOWER : Bound.UPPER]) return '0'
 
-    if (
-      tickSpaceLimits?.[Bound.LOWER] !== undefined &&
-      leftPrice &&
-      priceToClosestTick(leftPrice) <= tickSpaceLimits[Bound.LOWER]
-    ) {
-      return '0'
+    try {
+      if (
+        tickSpaceLimits?.[Bound.LOWER] !== undefined &&
+        leftPrice &&
+        priceToClosestTick(leftPrice) <= tickSpaceLimits[Bound.LOWER]
```

Comment:
not found crash with the desc link

and if this is issue, we should fixed it by add a division check but not just catch

for example: try check price's denominator !== 0